diff --git a/dist/internal/body-scroll-lock.svelte.js b/dist/internal/body-scroll-lock.svelte.js
index df6f86c4ff47ef828a1a4a8b6f00df8af2befd05..db2a7f75c3bf3c62b315701491aa7551811d528b 100644
--- a/dist/internal/body-scroll-lock.svelte.js
+++ b/dist/internal/body-scroll-lock.svelte.js
@@ -31,7 +31,7 @@ const bodyLockStackCount = new SharedState(() => {
     function resetBodyStyle(documentObj) {
         if (!BROWSER)
             return;
-        documentObj.body.setAttribute("style", initialBodyStyle ?? "");
+        documentObj.body.style.cssText = initialBodyStyle ?? "";
         documentObj.body.style.removeProperty("--scrollbar-width");
         isIOS && stopTouchMoveListener?.();
         // reset initialBodyStyle so next locker captures the correct styles
